projects/plutus/test/test_current_sense.c

```diff
@@ -0,0 +1,61 @@
+#include <inttypes.h>
+
+#include "current_sense.h"
+
+#include "critical_section.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+#include "ltc2484.h"
+#include "soft_timer.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+// Arbitrary number of testing samples
+#define TEST_NUM_SAMPLES 100
+
+static CurrentSenseStorage s_storage;
+
+static uint8_t s_callback_runs = 0;
+
+static void prv_callback(CurrentSenseValue *value, void *context) {
+  s_callback_runs++;
+  LOG_DEBUG("[%d / %d] | Voltage =  %" PRId32 ", Current =  %" PRId32 "\n", s_callback_runs,
```

Comment:
What is this supposed to test?

Reply:
This was just supposed to test that the samples were being taken.

Reply:
I mean the actual test. It doesn't seem to be verifying anything.

Reply:
Done